segments/filter/flowfilter/visitor.go

```diff
@@ -0,0 +1,502 @@
+// FROM https://github.com/bwNetFlow/flowfilter/
```

Comment:
But... why? This requires the flowpipeline project to have knowledge about the details of flowfilter's matching. That kinda goes against separation of concerns.